beacon-chain/rpc/statefetcher/fetcher.go

```diff
@@ -73,7 +93,7 @@ func (p *StateProvider) State(ctx context.Context, stateId []byte) (iface.Beacon
 			slotNumber, parseErr := strconv.ParseUint(stateIdString, 10, 64)
 			if parseErr != nil {
 				// ID format does not match any valid options.
-				return nil, errors.New("invalid state ID: " + stateIdString)
```

Comment:
I removed the appended ID because it is known to the requestor anyway, and it makes it easier to compare errors without it.